src/apiCalls.js

```diff
@@ -0,0 +1,5 @@
+export const fetchApiData = (type) => {
+  return fetch(`http://localhost:3001/api/v1/${type}`)
+    .then(response => response.json())
+    .catch(err => console.log("API error"))
```

Comment:
We're going to have to add error handling for this catch, so the User may know when an error has occurred.